OneSignalSDK/onesignal/core/src/main/java/com/onesignal/internal/OneSignalImp.kt

```diff
@@ -172,126 +173,160 @@ internal class OneSignalImp : IOneSignal, IServiceProvider {
 
         // do not do this again if already initialized
         if (isInitialized) {
```

Comment:
A  `if (isInitialized)` check was added in this PR in the new `synchronized` block, so we can remove this one.